platform/tss/tss_message_map_key.proto

```diff
@@ -4,16 +4,18 @@
  * ### Keywords
  * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
  * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
- * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119).
+ * document are to be interpreted as described in
+ * [RFC2119](https://www.ietf.org/rfc/rfc2119) and clarified in
+ * [RFC8174](https://www.ietf.org/rfc/rfc8174).
  */
 syntax = "proto3";
 
-package com.hedera.hapi.platform.tss;
+package com.hedera.hapi.node.state.tss;
```

Comment:
This should still be `platform.tss`, as the folders and packages should match in terminal elements.